content/companies/05-nutanix.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Nutanix
+excerpt: Know more about Nutanix's interview process ➡️
+index: 1
```

Comment:
make it 5

Reply:
Also seems like you committed `package-lock.json` by mistake. Kindly remove that?

Reply:
update the [README.md](https://github.com/kamleshchandnani/awesome-interview-process#-awesome-companies-list) section as well

Reply:
Oops yeah, I'll update it.